test/core/curlify.js

```diff
@@ -131,12 +132,35 @@ describe("curlify", function() {
             url: "http://example.com",
             method: "POST",
             headers: { "content-type": "multipart/form-data" },
-            body: "id=123&name=Sahar"
+            body: [
```

Comment:
I changed it to match the types which are actually passed from the UI.

Judging from the old code in swagger-ui 2.x the multipart form body was passed as a string.

In 3.x a multipart form body gets passed as this kind of structure:
```
Immutable.OrderedMap(
 1 -> Immutable.List("param1", "value1"),
 2 -> Immutable.List("param2", "value2"),
 ... and so on
)
```
I didn't want to change structure and risk breaking some other UI somewhere.